dashed/assets/javascripts/featured.js

```diff
@@ -1,6 +1,6 @@
 var $ = window.$ = require('jquery');
 var jQuery = window.jQuery = $;
-var px = require('./modules/dashed.js');
+require('./modules/dashed.js');
```

Comment:
do you need this? previously `px` contained the object returned by `./modules/dashed.js`, so now you have no handle to px, which probably means it's not called and this entire call is non necessary?